app/views/clients/viewclient.html

```diff
@@ -140,11 +140,11 @@ <h3 class="client-title">
 				<uib-tab heading="{{'label.heading.general' | translate}}">
                     <br>
                     <div class="container">
-						<div class="row">
+						<div class="row col-md-12 col-sm-12 ng-scope">
```

Comment:
1. Why you have used ng-scope here ? 
2. Why you have changed bootstrap ? It shifts in different line with your fix and more button open s on left side which makes difficult to view